nucleus/deployment/admin/src/main/java/org/glassfish/deployment/admin/DeployCommand.java

```diff
@@ -492,8 +492,6 @@ public void execute(AdminCommandContext context) {
             deploymentContext
                     = deployment.getBuilder(logger, this, report).
                             source(initialContext.getSource()).archiveHandler(archiveHandler).build(initialContext);
-            structuredTracing.register(deploymentContext);
```

Comment:
`structuredTracing` already registered as `initialContext` and `deploymentContext` are same instance.